repos/fdbt-site/src/pages/viewPassengerTypes.tsx

```diff
@@ -0,0 +1,278 @@
+                                                href="/viewPassengerTypes"
+                                            >
+                                                Edit
+                                            </a>
+                                        </li>
+
+                                        <li className="actions__item">
+                                            <a
+                                                className="govuk-link govuk-!-font-size-19 govuk-!-font-weight-regular actions__delete"
+                                                href="/viewPassengerTypes"
+                                            >
+                                                Delete
+                                            </a>
+                                        </li>
+                                    </ul>
+                                </div>
+
+                                <h4 className="govuk-!-padding-bottom-4">{sentenceCaseString(d.name)}</h4>
+
+                                <p className="govuk-body-s govuk-!-margin-bottom-2">
+                                    <span className="govuk-!-font-weight-bold">Max size:</span> {d.maxGroupSize}
+                                </p>
+
+                                {d.companions.map((c) => (
+                                    <p key={c.passengerType} className="govuk-body-s govuk-!-margin-bottom-2">
+                                        <span className="govuk-!-font-weight-bold">
+                                            {sentenceCaseString(c.passengerType)}:
+                                        </span>{' '}
+                                        {c.minNumber ? c.minNumber : '0'} - {c.maxNumber}
+                                    </p>
+                                ))}
+                            </div>
+                        </div>
+                    </div>
+                ))}
+            </div>
+
+            <button className="govuk-button" data-module="govuk-button">
+                Add a passenger group
+            </button>
+        </div>
+    );
+};
+
+const ViewPassengerTypes = ({ passengerTypes, passengerTypeGroups }: PassengerTypeProps): ReactElement => (
+    <BaseLayout title={title} description={description} showNavigation={true}>
+        <div className="govuk-width-container">
+            <main className="govuk-main-wrapper">
+                <div className="govuk-grid-row">
+                    <div className="govuk-grid-column-one-third">
+                        <div className="app-pane__subnav">
+                            <nav className="app-subnav" aria-labelledby="app-subnav-heading">
+                                <h2 className="govuk-visually-hidden" id="app-subnav-heading">
+                                    Pages in this section
+                                </h2>
+
+                                <ul className="app-subnav__section">
```

Comment:
Does this list belong to this page or is it shared with all pages in global settings?

Reply:
Should be shared, good shout! On it :)